packages/smart/funcs.go

```diff
@@ -517,7 +517,10 @@ func CreateTable(sc *SmartContract, name, columns, permissions string, applicati
 		default:
 			data = v.(map[string]interface{})
 		}
-		colname := converter.EscapeSQL(strings.ToLower(data[`name`].(string)))
```

Comment:
i think EscapeSQL should be outside checkColumnName function, because it should only CHECK something, but it replaces one string to another

Reply:
Fixed